modules/ROOT/pages/rest-sdk/edits/customize-the-connector.adoc

```diff
@@ -0,0 +1,21 @@
+= Customize Your Connector
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
```

Comment:
This is a reference topic, so I would title it Links to Connector Customization Information.